pnpm-lock.yaml

```diff
@@ -6,18 +6,9 @@ settings:
 
 catalogs:
   default:
-    '@ai-sdk/anthropic':
```

Comment:
I don't see any package.json changes, why the change here?

Reply:
lockfile rebuild maybe not sure why there are changes there

Reply:
Can we remove htem?

Reply:
@benjreinhart looking at it that's some weird bugged state, fixed it now